Doc/using/cmdline.rst

```diff
@@ -811,7 +811,7 @@ conflict.
 
    Defines the :data:`user base directory <site.USER_BASE>`, which is used to
    compute the path of the :data:`user site-packages directory <site.USER_SITE>`
-   and :ref:`Distutils installation paths <inst-alt-install-user>` for
+   and Distutils installation paths for
```

Comment:
Is it really useful to keep a reference to removed Distutils? setup.py is legacy, no?

Reply:
#108040 removes the reference, I wanted to keep each PR distinct. When the first one is merged I'll resolve the conflicts.

Reply:
Conflicts resolved.